frontend/app/player/MessageDistributor/managers/DOM/DOMManager.ts

```
@@ -372,20 +381,35 @@ export default class DOMManager extends ListWalker<Message> {
         //@ts-ignore
         vn.node.adoptedStyleSheets = [...vn.node.adoptedStyleSheets].filter(s => s !== styleSheet)
         return
+      case "load_font_face":
+        vn = this.vRoots.get(msg.parentID)
+        if (!vn) { logger.error("Node not found", msg); return }
+        if (vn instanceof VShadowRoot) { logger.error(`Node ${vn} expected to be a Document`, msg); return }
+        let descr: Object
+        try {
+          descr = JSON.parse(msg.descriptors)
+          descr = typeof descr === 'object' ? descr : undefined
+        } catch {}
```

empty catch; add console.error at least